tests/kw_config_loader_test.sh

```diff
@@ -12,6 +12,7 @@ function suite
   suite_addTest "parse_configuration_output_Test"
   suite_addTest "parse_configuration_standard_config_Test"
   suite_addTest "parse_configuration_files_loading_order_Test"
+  suite_addTest "test_show_variables"
```

Comment:
I thought we gonna drop `suite_addTest`...

Reply:
I think so too! But only after #312 is merged... The problem is that if a `suite` function is defined, shunit2 will not automatically run functions which start with `test`. I thought it was better to leave the refactoring of the entire test code to #312.